### ISSUE TITLE
File playlist

### ISSUE THREAD

**Issue by Cimbali** [Cimbali/pympress#9]
It would be nice to have a (possibly collapsible) list of files to open in the presenter window. Maybe with a file tree as well?
Depends on #8 

Useful in situations where a set of presentations are to be presented in order and the user would like to minimize the setup time between talks.

**Comment by complanar** [comment]
Dirty workaround:

Create a bash script or a terminal-one-liner, write all file paths in it and use it to open multiple “pympresses” one after another.

Something like:

    for file in "File1" "File2" "File3"; do
        pympress "$file"
    done

**Comment by Cimbali** [comment]
I’m not sure this would be very helpful in the end. Plus there’s the question of how to navigate between one file and the next.

Simply opening the file from the “Open” menu is not that hard or time consuming.

**Comment by Cimbali** [comment]
I don’t know if this is very generalisable, but in my recent experience people are more likely to change laptops between talks, rather than gathering all PDFs on a single machine. We probably should support that use case instead. Something like automatically putting the content window fullscreen on 2nd screen when we go from 1 to 2 monitors.

**Comment by complanar** [comment]
In the meantime (since 2018) I would also promote that. I use pympress in my classes and often switch between extended and cloned desktop because I need another software to show some things to my students.

When I switch to cloned desktop both windows of pympress move to the main monitor (that’s alright), but when I switch back to extended desktop the presentation view does not move back to the second screen. Because it is in fullscreen mode I can not simply drag it to the second screen. 

At the moment I drag the presenter window to the second monitor and then swap screens by pressing S. But this way the audience sees my notes and some other stuff for a moment and that seems not very professional to me … ;-)

Detecting these changes in screen setting, I think, is not completely the same thing as you described above, but maybe it is worth to keep in mind.

**Comment by Cimbali** [comment]
I think the even should trigger when we go from 1 to 2 monitors, regardless of whether we connect a cable or not, and “cloned” should appear as 1 monitor logically… I’ll test that situation as well.

**Comment by Cimbali** [comment]
So I have implemented the logic for this, that is able to manage screen connections/disconnections as well as cloning/extending, the issue I’m having is that window managers have their own ideas on where to place windows, and if we request positions too quick then out changes get overrided or overrided.

**Comment by Cimbali** [comment]
Auto-fullscreen on extending desktop (by connecting screen or through setting) now implemented. Only when “Start full screen” enabled.
